package.json

```diff
@@ -36,18 +36,16 @@
   },
   "license": "GPL-3.0-or-later",
   "dependencies": {
-    "@types/node": "^13.13.0",
     "debug": "^4.1.1",
-    "fs-extra": "^9.0.1",
     "lodash.pick": "^4.4.0",
     "napi-macros": "^2.0.0",
     "node-fetch": "^2.6.0",
     "node-gyp-build": "^4.1.0",
     "split2": "^3.1.1",
-    "tempy": "^0.3.0",
     "typescript": "^3.7.5"
   },
   "devDependencies": {
+    "@types/node": "^16.4.1",
```

Comment:
I had to bump the version because `fs/promise` is only available from Node.js 14 on. Is that a problem?

Reply:
Should I use
```suggestion
    "@types/node": "^14.0.0",
```
?

Reply:
Probably. Probably not. We definetly should make sure that we bump the nodejs versions on all cis scripts too. And see if everything still works :)

Reply:
> make sure that we bump the nodejs versions on all cis scripts too

Done, except for `.github/workflows/build-and-test-integration-linux.yml` where I don't understand how Node.js is installed.

Also, does it make sense to still run CI on Ubuntu 16.04?